### ISSUE TITLE
Annoying gray background in mutt 

### ISSUE THREAD

**Issue by nutilius** [isontheline/pro.webssh.net#932]
# Bug description
When mutt client has settings like
`color normal white black `
what means 'display white characters on black background' 
terminal display white characters on grey background (but only to the end of printed line, the rest is black).

I've seen this on Linux and NetBSD.

# Screenshots

<img width="713" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/17044767/17472cd0-25f2-44c2-8ad8-b70aa153ebb9">

# Steps to reproduce
1. Start mutt without configuration file - make sure that something is in the mailbox and will be presented in mutt index.
2. Entries should be presented as  white on black characters
3. Set command -  press `:` (colon)  and in command line type `color normal white black` 
4. Entries in index will change background to gray.

**Comment by isontheline** [comment]
Hello @nutilius 👋

Thank you so much for these reproduce steps 🙏

I will try to find the root cause inside WebSSH or the xterm.js library

Have a nice day ☀️

**Comment by isontheline** [comment]
I've followed your steps but I'm not able to reproduce :

<img width="672" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/bcb7bfbb-074c-4f60-8a75-a79ea2ef2d09">

**Comment by nutilius** [comment]
But I see gray background, not black, why?
I've observed the effect with pure black background.

**Comment by isontheline** [comment]
When I put red background color it applied as expected.

I think black is default color and as I use gray as my terminal bg color it display as black.

Have you tried to change your terminal BG color to black instead of gray?

**Comment by nutilius** [comment]
I think it is the difference. I have by default black background. 
My session after connection looks like below:

<img width="1012" alt="Zrzut ekranu 2023-07-6 o 11 04 30" src="https://github.com/isontheline/pro.webssh.net/assets/17044767/39b11a9c-732e-4a0d-8c4e-8d40605813bd">

After running mutt (without any settings in muttrc) I see something like this:

<img width="1012" alt="Zrzut ekranu 2023-07-6 o 11 01 22" src="https://github.com/isontheline/pro.webssh.net/assets/17044767/6b232203-df84-43d7-980e-cc3aca4df801">

As you see lines are filled with gray background in place where characters were put on the screen.
Because in your settings gray is default backgroun (what I always change to black), I suspect, that you leave somewere your "default gray" color, not default color set by user.

**Comment by nutilius** [comment]
The settings for session:

<img width="1001" alt="Zrzut ekranu 2023-07-6 o 11 13 31" src="https://github.com/isontheline/pro.webssh.net/assets/17044767/d0874d18-de8e-44e0-a0cf-818a2160eb21">

**Comment by nutilius** [comment]
One thing yet - when I change background to f.ex. green, I see the same efect on entries in mutt:
<img width="1007" alt="Zrzut ekranu 2023-07-6 o 11 16 52" src="https://github.com/isontheline/pro.webssh.net/assets/17044767/10054f81-7c29-4ca8-8c6a-919ef25480ca">

so definitly the gray color is "fixed" in some place.

**Comment by isontheline** [comment]
You're right seems to be fixed but I think it's more related to xterm.js library.
I'm checking if I can customise a setting.

**Comment by isontheline** [comment]
I don't have found, yet

I also need to [upgrade Xterm.js](https://github.com/isontheline/pro.webssh.net/issues/912) but I don't know if it will resolve.

**Comment by isontheline** [comment]
I will upgrade Xterm.js to 5.2 due to : https://github.com/xtermjs/xterm.js/issues/4579

**Comment by isontheline** [comment]
A new WebSSH release will come in a few hours.

I hope it will fix this issue due to https://github.com/xtermjs/xterm.js/issues/4579

**Comment by isontheline** [comment]
Hello @nutilius 👋

Could you try to change TERM type to « xterm-color » inside your connection > Terminal Settings ?

Thank you

**Comment by nutilius** [comment]
I've changed to xterm-color and even to xterm - nothing changed. But remember that I still test on last available version - 22.4.1127 - generated 10-07-2023

**Comment by nutilius** [comment]
OK

**Comment by nutilius** [comment]
Feedback sent via TestFlight - unfortunately problem still exists.

**Comment by isontheline** [comment]
What’s the output of : 

`echo $TERM`

**Comment by nutilius** [comment]
For terminal type XTERM-COLOR is (of course) xterm-color (small letters).
For terminal type XTERM-256COLOR is  xterm-256color (small letters).

**Comment by isontheline** [comment]
Thanks for sharing screenshots on TestFlight feedback'

With "xterm-color" the result is better but still have a non-black color as you said
